app.rb

```diff
@@ -205,14 +189,24 @@ def handle_wrong_input
 
 def main
   loop do
-    prompt_main
+    puts ''
+    puts '=================================================='
+    puts '=   PLEASE CHOOSE AN OPTION BY TYPING A NUMBER   ='
+    puts '=================================================='
+    puts ''
+    puts '  1 - List of all books'
+    puts '  2 - List of all people'
+    puts '  3 - Create a person'
+    puts '  4 - Create a book'
+    puts '  5 - Create a rental'
+    puts '  6 - List od all rentals for a given id'
+    puts '  7 - Exit'
+    print '  :> '
     input = gets.chomp.to_i
     handle_case(input)
     puts ''
     puts '**************************************************'
   end
-  puts ''
-  puts ''
+  puts '\n\n'
 end
-
 main
```

Comment:
- [ ] Great work but I guess there is confusion between you and the previous reviewers because previous reviewers asked you to put this method into a file name main.rb file which will work as an entry point file 🤔 Kindly make a file main.rb and put this method into that file so user can start the app using `ruby main.rb` command 👍

Reply:
can I rename the app.rb to main.rb?